packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.js

```diff
@@ -0,0 +1,125 @@
+import asyncStates from '@zooniverse/async-states'
+import { flow, getEnv, types } from 'mobx-state-tree'
+
+const CONSENSUS_SCORE_TO_RETIRE = 3
+const MINIMUM_VIEW_TO_RETIRE = 5
+const REDUCER_KEY = 'alice'
+
+const TranscriptionLine = types.model({
+  consensusReached: types.boolean,
+  frame: types.number,
+  hasCollaborated: types.boolean,
+  points: types.array(types.frozen({})),
+  previousAnnotation: types.boolean,
+  textOptions: types.array(types.string),
+})
+
+const TranscriptionReductions = types
+  .model('TranscriptionReductions', {
+    caesarReducerKey: types.frozen(REDUCER_KEY),
```

Comment:
Is there a way to set up `types.literal` so that you don’t have to explicitly state the reducer key in snapshots? I used frozen because it’s suggested for setting default values.

Reply:
I want subjects to be able to run
```js
TranscriptionReductions.create({ subjectId, workflowId })
```
and have the reducer key set by the model, which isn't possible with `types.literal()`.

Reply:
I deleted this, since it's a constant now.